extension/js/common/core/crypto/pgp/openpgp-key.ts

```diff
@@ -194,40 +182,25 @@ export class OpenPGPKey {
    * @param keyToUpdate - an existing Key object to update, optional. Useful in encryptKey and decryptKey, because the operation
    *    is done on the original supplied object.
    */
-  public static convertExternalLibraryObjToKey = async (opgpKey: OpenPGP.key.Key, keyToUpdate?: Key): Promise<Key> => {
+  public static convertExternalLibraryObjToKey = async (opgpKey: OpenPGP.Key, keyToUpdate?: Key): Promise<Key> => {
     const { isPrimaryKeyStrong, keyWithoutWeakPackets } = OpenPGPKey.removeWeakKeyPackets(opgpKey);
```

Comment:
I suppose there was new functionality for this in OpenPGP.js v5, where we can simply configure which algorithms to use or forbid? Either in global config or when doing particular actions (forgot about their implementation). In which case we could stop using `removeWeakKeyPackets` / remove it. Can be another issue.

Reply:
https://github.com/FlowCrypt/flowcrypt-browser/issues/4905